### DISCUSSION TITLE
Suggest adding suffixes to all numbers

### DISCUSSION THREAD

**Discussion by nczsl** [dotnet/csharplang#8063] [General]
At present, there are four types of numbers, including byte/sbyte/short/ushort, that do not have suffixes, which often brings unnecessary trouble to work. It is recommended to also add suffixes to these types of numbers

**Comment by KennethHoff** [comment]
There are tons of number types, and .Net 9 adds at least 3 more; Decimal32, Decimal64, and Decimal128. It's not sustainable to add suffixes
